integrations/pgvector-documentstore.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: integration
+name: pgvector
+description: A Document Store for storing and retrieval from pgvector
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/pgvector-haystack/
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/pgvector
+type: Document Store
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
+version: Haystack 2.0
```

Comment:
Looks like the logo is missing?

Reply:
pgvector does not have a logo. Maybe because it is an extension of PostgreSQL.
We could also insert the logo of PostgreSQL, but it would be a bit misleading